docs/source/ko/_toctree.yml

```
@@ -20,8 +20,8 @@
       title: (번역 중) Repository
     - local: in_translation
       title: (번역 중) Search
-    - local: in_translation
-      title: (번역 중) Inference
+    - local: inference
```

```suggestion
    - local: guides/inference
```
toctree 내 path가 누락되어있어 수정하였습니다.